Integrations/integration-Giphy.yml

```diff
@@ -0,0 +1,80 @@
+description: Display random GIF in the War Room (e.g. !giphy hello)
+detaileddescription: 'You can get API Key from: https://github.com/Giphy/GiphyAPI#access-and-api-keys'
+configuration:
+- display: Giphy URL
+  name: url
+  defaultvalue: http://api.giphy.com/v1/gifs/random
+  type: 0
+  required: true
+- display: API Key
+  name: apikey
+  defaultvalue: ""
```

Comment:
Do you want to add the default test api key ?
Not sure about it, but than users can use it easily :)